azure-quantum/azure/quantum/job.py

```diff
@@ -82,3 +83,9 @@ def get_results(self):
 
         result = json.loads(payload.decode('utf8'))
         return result
+            
+    @staticmethod
+    def create_job_id() -> str:
+        """Create a unique id for a new job.
+        """
+        return uuid.uuid1()
```

Comment:
Strictly speaking, `uuid.uuid()` returns a `UUID` object. If you want this to return a string as the type hint suggests I would change this to 
```suggestion
        return str(uuid.uuid1())
```